google/credential/tool.gpt

```diff
@@ -21,6 +21,8 @@ Tools: ../../oauth2
             "https://mail.google.com/",
             "https://www.googleapis.com/auth/gmail.readonly",
             "https://www.googleapis.com/auth/gmail.compose",
+            "https://www.googleapis.com/auth/webmasters",
+            "https://www.googleapis.com/auth/webmasters.readonly",
```

Comment:
Does adding these scopes break existing credentials?

Reply:
IIUC, they'll at least need to be added to the [UI's instructions for configuring the oauth app](https://github.com/obot-platform/obot/blob/4e0b2562481354097066610b8e521b84f7a61f13/ui/admin/app/lib/model/oauthApps/providers/google.ts#L16)

Reply:
If my understanding is correct, if an existing credential was authenticated for a tool that was updated to require a new scope would see an unauthorized response. They would need to delete the credential and reauthenticate. 

In this case, we are adding an entirely new tool / bundle so it would only be impactful if it was added to an existing agent and picked up in the same thread.